Makefile

```diff
@@ -0,0 +1,48 @@
+
+GODOT_BIN_PATH = ../godot_fork/bin/godot.x11.tools.64.llvm
+HEADERS = ../godot_headers
+TARGET = debug
+NAME = godot-cpp
+
+BASE = scons n=$(NAME) generate_bindings=yes target=$(TARGET) headers=$(HEADERS) godotbinpath=$(GODOT_BIN_PATH) -j4
```

Comment:
Sometimes it's desirable to not re-generate the bindings, maybe this can be a flag too?

Talking about this, it might be good to change the way the bindings are generated so Scons is more aware of them when recompiling. I'll open an issue for that.

Reply:
Yeah, I've seen that todo in scons.
Shouldn't generate_bindings be `yes` by default? Modules won't work without bindings, right?

Reply:
We could check if there's no bindings - generate them. And replace flag `generate_bidings` with `regenerate_bidings` so  you can force it to regenerate them.